bin/node/pallets/pallet-updater/src/tests.rs

```diff
@@ -0,0 +1,254 @@
+use super::*;
+use frame_support::{assert_noop, assert_ok, parameter_types};
+use frame_system as system;
+use sp_core::H256;
+use sp_runtime::{
+	testing::Header,
+	traits::{BlakeTwo256, IdentityLookup},
+	BuildStorage,
+};
+
+use crate::{self as pallet_updater, Config};
+
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+// Configure a mock runtime to test the pallet.
+frame_support::construct_runtime!(
```

Comment:
`construct_runtime!` along with the config implementations should be in the `mock.rs` file. We should always follow the structure of others pallets.